docs/components/movement-sensor/imu/imu-vectornav.md

```diff
@@ -76,10 +77,10 @@ Click **Create Component** and then fill in the attributes for your model.
 
 ## Attributes
 
-Name | Type | Default Value | Description
------ | ----- | ----- | -----
-`board` | string | - | The name of the board to which the device is wired
-`spi` | string | - | The name of the SPI bus over which the device communicates with the board. On a Raspberry Pi, people often use the bus named "1."

```

Comment:
I feel like this is actually useful info? I know we don't want to be Pi-specific, but is this info elsewhere?

Reply:
ah ok!